### PR TITLE
Remove create/delete oneoffs

### PR DESCRIPTION
**Pull request by milescrabill**
@jvehent r+?

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -65,14 +65,20 @@ Each module is documented in its own modules/<modulename> directory.
...
+* `Reset` indicates whether user reset is enabled
+* `ResetUsers` contains a comma separated list of users to reset
+* `UidMap` is a mapping of UIDs as described in the next section
+* `Credentials` contains module-specific credentials
+* `Parameters` contains module-specific parameters
+Notify
+* `ApplyChanges` indicates whether to apply changes or dryrun
+* `NotifyUsers` indicates whether notifications are enabled
+* `LdapCli` is a mozldap.Client
+* `Debug` indicates whether debug messages should be printed
```
Comment (jvehent) [inline Readme.md]: Not sure why you capitalized those and added the stuff under Notify. This section describes the configuration file, not the internal struct of `userplex/modules`.

File: main.go [hunk 2]
```diff
@@ -189,12 +187,20 @@ func run(conf conf) {
 			log.Printf("[warning] %s module not registered, skipping it", modconf.Name)
 			continue
 		}
+		// one-off reset uses the first module config, skips subsequent
+		if *resetUsers != "" {
+			if _, ok := moduleHasRun[modconf.Name]; ok {
+				log.Printf("[warning] SKIPPING duplicate module config %s! Reset uses only the FIRST configuration for a given module.", modconf.Name)
```
Comment (jvehent) [inline main.go:193]: only print this if the conf has multiple modules of the same name

File: Readme.md (hunk 1 above)
Comment (milescrabill) [reply Readme.md]: Yeah, oops. I misinterpreted it as for module authors, not users writing config. Also that Notify was a mistake. Fixing!

File: main.go (hunk 2 above)
Comment (milescrabill) [reply main.go:193]: that's what this does

**Pull request merged**